src/main/java/com/lucas/plugintest/KickBlockCommand.java

```diff
@@ -49,8 +49,7 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
             //expulsar
             target.kickPlayer("voce foi expulso pelo " + player.getDisplayName() + " pelo motivo: " + reason);
             player.sendMessage("Voce kickou o jogador " + target.getName());
-            return true;
-        }
+            return true;        }
```

Comment:
Indent correctly }, as it is closing the function `onCommand()`, it is a good practice to indent it according to the function's indention

Reply:
thanks, ill correctly =)